fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -1815,7 +1817,34 @@ private Partition createPartitionWithIndices(String clusterName, long dbId, long
             }
 
             if (!ok || !countDownLatch.getStatus().ok()) {
-                errMsg = "Failed to create partition[" + partitionName + "]. Timeout:" + (timeout / 1000) + " seconds.";
+                SystemInfoService infoService = Env.getCurrentSystemInfo();
+                List<String> allBEHost = countDownLatch.getLeftMarks().stream().map(item -> {
+                    return infoService.getBackend(item.getKey()).getHost();
+                }).distinct().collect(Collectors.toList());
+                List<String> downBEList = countDownLatch.getLeftMarks().stream().map(item -> {
+                    return infoService.getBackend(item.getKey());
+                }).distinct().filter(new Predicate<Backend>() {
+                    @Override
+                    public boolean test(Backend backend) {
+                        String ip = "http://" + backend.getHost() + ":" + backend.getBePort() + "/api/health";
+                        try {
+                            HttpUtils.doGet(ip, null);
+                        } catch (Exception e) {
+                            return true;
+                        }
+                        return false;
+                    }
```

Comment:
Maybe we could make use of https://github.com/apache/doris/blob/2dddab03a1a0c73325159c31f5c23653ad63f69c/fe/fe-core/src/main/java/org/apache/doris/system/SystemInfoService.java#L368-L383 
to check alive BE and no need to ping here.